src/config.cc

```diff
@@ -271,8 +310,44 @@ load_binding_settings(en_config_type config_type) {
     return loaded_bindings;
 }
 
+audio_settings
+load_audio_settings(const en_config_type & config_type) {
+    audio_settings loaded_audio;
+    config_t cfg{};
+    config_setting_t *audio_config_setting = nullptr;
+
+    const char* config_path = get_audio_config_path(config_type);
+
+    config_init(&cfg);
+
+    if (!config_read_file(&cfg, config_path))
+    {
+        printf("%s:%d - %s reading %s\n", config_error_file(&cfg),
+               config_error_line(&cfg), config_error_text(&cfg), config_path);
+        config_destroy(&cfg);
+
+        return loaded_audio;
+    }
+
+    audio_config_setting = config_lookup(&cfg, "audio");
+
+    if (audio_config_setting != nullptr) {
```

Comment:
I've been doing mostly C coding lately, is this preferred over if (audio_config_setting) ?